### ISSUE TITLE
Empty PAF

### ISSUE THREAD

**Issue by schorlton** [alexomics/read-paf#7]
Hi @alexomics , 

Thanks again for the great tool!

I hit a bug where I had an empty PAF (no read alignments, this seems to be valid output from minimap2?). Tried reading this in with `parse_paf` into a pandas dataframe but fails (I believe because of the tag columns). Is it possible to produce an empty df? Should this be the desired behaviour?

To reproduce:
```bash
touch test.paf
```

```python
from readpaf import parse_paf

with open("test.paf", "r") as handle:
    df = parse_paf(handle, dataframe=True)
```

Thanks!

**Comment by alexomics** [comment]
I've added a branch [`issue7`](https://github.com/alexomics/read-paf/tree/issue7) that has a test/fix for this. You should be able to install this version (0.0.10a1) using:

```bash
pip install --upgrade git+https://github.com/alexomics/read-paf@issue7
```

Let me know if that works 😊

**Comment by alexomics** [comment]
There may also be a better way of checking if the DataFrame is empty, see: https://stackoverflow.com/a/24933234. Though it's worth considering what is empty? E.g: no rows and named columns?

**Comment by schorlton** [comment]
Wow, thanks for such as fast response and fix! I wonder if the columns of the df should be the standard 12 PAF columns (same as if there were alignments), with just no rows as you allude to above? Currently, I'm getting just an empty df with no columns.

**Comment by schorlton** [comment]
Hi @alexomics , just wondering if you could point me in the easiest way to retain the field headers of the df even if the PAF is empty? Thanks!

**Comment by alexomics** [comment]
The easiest way is for me to return the DataFrame with the column names already there 😅

If you give v0.0.10a2 a go let me know if that works or has any unexpected behaviour. I'll try to get this on to PyPI ASAP!

**Comment by schorlton** [comment]
Wow, thanks so much! Gave it a go and it's great!

**Comment by alexomics** [comment]
I've published this as v0.0.10. Thanks for testing it out!
